torch/onnx/_internal/exporter.py

```diff
@@ -809,7 +823,7 @@ def fake_context(self) -> Optional[ONNXFakeContext]:
     def adapt_torch_inputs_to_onnx(
         self,
         *model_args,
-        model: Optional[
+        model_with_state_dict: Optional[
```

Comment:
we need the `model`, because the lifted "constant tensors" are not part of the `ExportedProgram.state_dict` (?!?!?!)

I will investigate whether I can make that change on `ExportedProgram.state_dict` , simplifying the requirement to `state_dict` only (which is the ideal case)